cron/k8s/worker.yaml

```diff
@@ -42,6 +42,8 @@ spec:
           value: "gs://ossf-scorecards-cache"
         resources:
           requests:
-            memory: 1Gi
+            cpu: 3
```

Comment:
can go higher, at this point, we want stability, so can try 5 with 5gi here and limit can be even higher like 10 or 15 or whatever.

Reply:
Done.